infrastructure/general/setup-full.md

```diff
@@ -0,0 +1,366 @@
+    ```bash
+    rustup show
+
+    # You should see output similar to:
+
+    active toolchain
+    ----------------
+
+    stable-aarch64-apple-darwin (default)
+    rustc 1.68.1 (8460ca823 2023-03-20)
+    ```
+
+    ```bash
+    rustup +nightly show
+
+    # You should see output similar to:
+
+    installed targets for active toolchain
+    --------------------------------------
+
+    aarch64-apple-darwin
+    wasm32-unknown-unknown
+
+    active toolchain
+    ----------------
+
+    nightly-aarch64-apple-darwin (overridden by +toolchain on the command line)
+    rustc 1.71.0-nightly (9ecda8de8 2023-04-30)
+    ```
+
+    - Once Rust is configured, run the following command to clone and build the Polkadot code:
+    ```bash
+    git clone https://github.com/paritytech/polkadot-sdk polkadot-sdk
+    cd polkadot-sdk
+    ./scripts/init.sh
+    cargo build --release
+    ```
+    - Start your node:
+
+    ```bash
+    polkadot --name "Your Node's Name"
+    ```
+
+    - Find your node on [Telemetry](https://telemetry.polkadot.io/#list/Polkadot)
+
+=== "Windows"
+    - Install [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+    - Install [Ubuntu](https://docs.microsoft.com/en-us/windows/wsl/install-win10) (same webpage).
+    - Determine the latest version of the
+    [Polkadot binary](https://github.com/paritytech/polkadot-sdk/tree/master/polkadot/releases).
+    - Download the correct Polkadot binary within Ubuntu by running the following command. Replace
+    `*VERSION*` with the tag of the latest version from the last step (for example, `v0.8.22`):
+    ```bash
+    curl -sL https://github.com/paritytech/polkadot-sdk/releases/download/*VERSION*/polkadot -o polkadot
+    ```
+
+    - Then, run the following:
+    ```bash
+    sudo chmod +x polkadot
+    ```
+    - Start your node:
+    ```bash
+    ./polkadot --name "Your Node's Name"
+    ```
+
+    - Find your node on [Telemetry](https://telemetry.polkadot.io/#list/Polkadot){target=_blank}
+
+=== "Linux (Standalone)"
+
+    !!!info
+        The nature of pre-built binaries means that they may not work on your particular architecture or
+        Linux distribution. If you see an error like `cannot execute binary file: Exec format error` it
+        likely means the binary is not compatible with your system. You will either need to compile the
+        [**source code**](#clone-and-build) or use [**Docker**](#using-docker).
+
+    - Determine the latest version of the
+    [Polkadot binary](https://github.com/paritytech/polkadot-sdk/releases).
+
+    - Download the correct Polkadot binary within Ubuntu by running the following command. Replace
+    `*VERSION*` with the tag of the latest version from the last step (e.g. `v0.8.22`):
+
+    ```bash
+    curl -sL https://github.com/paritytech/polkadot-sdk/releases/download/*VERSION*/polkadot -o polkadot
+    ```
+
+    - Run the following: `sudo chmod +x polkadot`
+    - Run the following:
```

Comment:
This may be nitpicking, but I'd try to avoid this titles since are not super descriptive of the command that the user will run, maybe:

"Provide execution rights to the polkadot binary:"
```
sudo chmod +x polkadot
```